deployment/deployment.go

```diff
@@ -331,3 +339,36 @@ func getDeployment(deploymentId string, client *houston.Client) (*houston.Deploy
 
 	return &r.Data.GetDeployment, nil
 }
+
+func meetsAirflowUpgradeReqs(airflowVersion string, desiredAirflowVersion string) error {
+	upgradeVersion := strconv.FormatUint(settings.NewAirflowVersion, 10)
+	minRequiredVersion := "1.10.14"
+	airflowUpgradeVersion, err := semver.NewVersion(upgradeVersion)
+	if err != nil {
+		return err
+	}
+
+	desiredVersion, err := semver.NewVersion(desiredAirflowVersion)
+	if err != nil {
+		return err
```

Comment:
@andriisoldatenko I can add more tests for these cases if you think we need them.  I can put it invalid version numbers.

Reply:
it's okey.